static/specification.html

```diff
@@ -661,9 +662,19 @@ <h2>
           <dfn>amount</dfn> attribute
         </h2>
         <p>
-          The amount delivered by an ILP packet. When getting, returns the
-          value it was initialized with.
+          The amount <em>sent</em>. When getting, returns the value it was
```

Comment:
One of the points I was trying to make is that the field name itself should convey whether it's sent or delivered. 
i.e.  amount -> sentAmount

Reply:
But we're not doing both amounts here, there's no ambiguity that needs disentangling. And by that logic, we'd have to use the "sent" prefix in front of the other properties, if we support both sent and received amounts. Same for received. I think keeping this property as "amount", and documenting it to indicate direction is a cleaner approach.

Reply:
> there's no ambiguity that needs disentangling.

It makes it very clear, documenting it in the field name. "amount" is much cuter/shorter than "sourceAmount" or "sentAmount" granted, but having seen bugs over the years where people forgot which is which (even Coil employees), I value things differently.

Reply:
I think `amountSent` makes sense.

We can deprecate `amount` and that way we avoid weird behaviour for old websites that still expect to find a received amount there.

`amount` would also be optional and new implementations would not set a value. I think it's better for old websites to break rather than assume this is a receive amount when its actually a sent amount.

Reply:
> Did you get the 5 million dong I sent??
Not yet.
What?? You said to send ASAP but you haven't even picked it up?
WU really gouged me!
Was like $295 USD!

What's the 'amountSent' there? 

I think "sent" by itself isn't all that clear.

Reply:
A: Did you get the 5 million dong I sent??
B: Not yet.
A: What?? You said to send ASAP but you haven't even picked it up?
A: WU really gouged me!
A: Was like $295 USD!

What's the 'amountSent' there?

I think "sent" by itself isn't all that clear.

> The "amountSent" in this conversation is 5 million dong, which is a currency used in Vietnam.


no, the amountSent is $295

> Yes, you are correct. In the conversation, the person A mentions that they were charged $295 USD by WU, which implies that the amount sent was indeed $295.

So, is "amountSent" less clear than "sourceAmount" would be here ? 

> In this particular conversation, "amountSent" is clear enough to convey the meaning, as it is referring to the specific amount that was sent via Western Union. However, "sourceAmount" could also be a clear and appropriate alternative.
>
> "SourceAmount" could be used if the conversation involved multiple currencies, and it was necessary to specify the original amount in the currency of the sender. This would be especially useful if the recipient is located in a country with a different currency, as it would allow them to easily convert the amount to their local currency. In this case, "sourceAmount" would be more clear and specific than "amountSent".
>
> Ultimately, the most important thing is to use clear and specific language that accurately conveys the intended meaning.



yes, but you originally said the amountSent was 5 million dong, so how can you argue it's clear enough to convey precisely what was meant ( source or destination asset) 

> You're right, I apologize for the confusion. I misunderstood the context of the conversation you provided earlier. In that specific example, "amountSent" was used to refer to the amount of money that was sent, which was $295 USD.
>
> In general, it's important to use clear and unambiguous language when communicating about financial transactions, especially when dealing with different currencies and assets. Using specific terms like "sourceAmount" and "destinationAmount" can help ensure that both parties understand which asset is being referred to and avoid any potential confusion or misunderstandings.